### PR TITLE
Opened PR

### PR DESCRIPTION
**Pull request by tatapuchi**

### CODE REVIEW

File: Idle.sln [hunk 1]
```diff
@@ -19,7 +19,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Idle.Logic", "src\Idle.Logi
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Idle.Views", "src\Idle.Views\Idle.Views.csproj", "{BE1E9528-772F-433F-B67E-49F9DD7577F5}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Idle.Framework", "src\Idle.Framework\Idle.Framework.csproj", "{9440FF3B-4CAC-49FD-AC38-DC0FC2C23089}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Idle.Common", "src\Idle.Framework\Idle.Common.csproj", "{9440FF3B-4CAC-49FD-AC38-DC0FC2C23089}"
```
Comment (BrayanKhosravian) [inline Idle.sln:22]: i still see `Idle.Framework`

File: src/Idle.DataAccess/Idle.DataAccess.csproj [hunk 2]
```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Idle.Framework\Idle.Framework.csproj" />
```
Comment (BrayanKhosravian) [inline src/Idle.DataAccess/Idle.DataAccess.csproj:17]: same here

**Review by BrayanKhosravian — COMMENTED** [review]
wrong branch. make a new branch and cherry pick the commits then make a new pr.

**Comment by tatapuchi** [comment]
I'll fix the commits once im back from school tmmrw

File: src/Idle.DataAccess/Idle.DataAccess.csproj (hunk 2 above)
Comment (tatapuchi) [reply src/Idle.DataAccess/Idle.DataAccess.csproj:17]: doesnt that mean it has been removed

**Pull request closed without merging**